pkg/engine/transform/transformtestfiles/dcos_template.json

```diff
@@ -1,1140 +0,0 @@
-{
```

Comment:
Why did this file and `dcos_scale_template.json` have to be removed? Some of AKS Engine is still in use by the ACS service and we need to ensure any relevant unit tests are still passing.

Reply:
only the test TestNormalizeForVMSSScaling() uses this file, and the function has been deleted with my change. So I guess this file is not needed anymore?